RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs

```diff
@@ -60,21 +63,31 @@ public AddTestModuleCommand(IVBE vbe, RubberduckParserState state, IGeneralConfi
 
         private const string TestModuleBaseName = "TestModule";
 
+        private const string EarlyBinding = "New {0}";
+        private const string LateBinding = "CreateObject(\"{0}\")";
+
         private string GetTestModule(IUnitTestSettings settings)
         {
-            var assertClass = settings.AssertMode == AssertMode.StrictAssert ? string.Empty : "Permissive";
-            var moduleBinding = settings.BindingMode == BindingMode.EarlyBinding
-                ? string.Format(ModuleEarlyBinding, assertClass)
-                : ModuleLateBinding;
+            bool isEarlyBinding = settings.BindingMode == BindingMode.EarlyBinding;
+
+            var assertType = string.Format("Rubberduck.{0}AssertClass", settings.AssertMode == AssertMode.StrictAssert ? string.Empty : "Permissive");
+
```

Comment:
the `assertType` and `fakesType` assignments shouldn't be separated by an empty line here.